src/meta/src/hummock/compaction_group/mod.rs

```diff
@@ -155,21 +153,25 @@ impl MetadataModel for CompactionGroup {
 // directly fetch such options from catalog when creating compaction jobs.
 #[derive(Clone, Debug, PartialEq, Default)]
 pub struct TableOption {
-    ttl: u32,
+    ttl: Option<u32>,
 }
 
 impl From<&risingwave_pb::hummock::TableOption> for TableOption {
     fn from(table_option: &risingwave_pb::hummock::TableOption) -> Self {
-        Self {
-            ttl: table_option.ttl,
-        }
+        let ttl = if table_option.ttl == 0 {
```

Comment:
Use a const instead of `0`.

Reply:
fix it with a constant value 

i gonna unifty the constant value define in other pr (in constant::hummock mod)